packages/ng-form/src/lib/components/form/form.component.html

```diff
@@ -0,0 +1,11 @@
+<form
+  [formGroup]="form"
+  [class]="formStyleClass"
+  (ngFormSubmit)="submitFormSave()">
```

Comment:
Not sure this is the way 🤔 why don't we let the use do what he needs to do with the `<form>` tag? Is there a reason for us to bring it within this library? What I expected were directives to be used within our applications.